content/en/docs/concepts/glossary.md

```diff
@@ -9,189 +9,369 @@ others. This page captures terminology used in the project and what it means.
-- **[REST](https://en.wikipedia.org/wiki/Representational_state_transfer):**
-  Representation State Transfer.
-- **[RPC](https://en.wikipedia.org/wiki/Remote_procedure_call):** Remote
-  Procedure Call.
-- **[Sampling]({{< relref "/docs/reference/specification/trace/sdk#sampling" >}}):**
-  A mechanism to control the amount of data exported. Most commonly used with
-  the `Tracing` `Data Source`.
-- **SDK:** Software Development Kit. Refers to a telemetry SDK that denotes a
-  `Library` that implement the OpenTelemetry `API`.
-- **Semantic Conventions:** Defines standard names and values of `Metadata` in
-  order to provide vendor-agnostic telemetry data.
-- **Service:** A component of an `Application`. Multiple instances of a
-  `Service` are typically deployed for high availability and scalability. A
-  `Service` may be deployed in multiple locations.
-- **[Span]({{< relref "/docs/reference/specification/trace/api#span" >}}):**
-  Represents a single operation within a `Trace`.
-- **Span Link:** A span link is a link between causally-related spans. For
-  details see [Links between
-  spans]({{< relref "/docs/reference/specification/overview#links-between-spans" >}})
-  and [Specifying
-  Links]({{< relref "/docs/reference/specification/trace/api#specifying-links" >}}).
-- **[Specification](/docs/concepts/components/#specification):** Describes the
-  cross-language requirements and expectations for all implementations.
-- **[Status]({{< relref "/docs/reference/specification/trace/api#set-status" >}}):**
-  The result of the operation. Typically used to indicate whether an error
-  occurred.
-- **Tag:** see `Metadata`.
-- **[Trace]({{< relref "/docs/reference/specification/overview#traces" >}}):** A
-  `DAG` of `Spans`, where the edges between `Spans` are defined as parent/child
-  relationship.
-- **[Tracer]({{< relref "/docs/reference/specification/trace/api#tracer" >}}):**
-  Responsible for creating `Spans`.
-- **Transaction:** See `Distributed Tracing`.
-- **[zPages][]:** An in-process alternative to external exporters. When
-  included, they collect and aggregate tracing and metrics information in the
-  background; this data is served on web pages when requested.
+### **Aggregation**
+
+The process of combining multiple measurements into exact or estimated
+statistics about the measurements that took place during an interval of time,
+during program execution. Used by the [`Metric`](#metric)
+[`Data Source`](#data-source).
+
+### **API**
+
+Application Programming Interface. In the OpenTelemetry project, used to define
+how telemetry data is generated per [`Data Source`](#data-source).
+
+### **Application**
+
+One or more [`Services`](#service) designed for end users or other applications.
+
+### **APM**
+
+Application Performance Monitoring. Typically a back-end of the
+[`Tracing`](#trace) [`Data Source`](#data-source).
+
+### **[Attribute][]**
+
+A key-value pair. Used by the [`Tracing`](#trace) [`Data Source`](#data-source)
```

Comment:
This seems outdated. Attributes can be used across telemetry signals (e.g. metrics also use Attributes as the term instead of Labels).

Reply:
Fixed